Documentation/Testing/ExtensionTesting.rst

```diff
@@ -2,17 +2,24 @@
 .. index:: pair: Testing; Extensions
 .. _testing-extensions:
 
+
+
 =================
 Extension testing
 =================
 
-..  attention::
-    The example in this chapter covers only outdated TYPO3 and PHP versions. Help us bring
-    it up to date: See :ref:`h2document:contribute`.
+.. attention::
+
+    This page has not been fully updated yet. Specifically the following:
```

Comment:
I believe this does not require a warning. Just saying that we cover one version testing here should be enough

Reply:
Have removed the note now. (Under "Scope" it is already explained that we test only one core version).